package.json

```diff
@@ -209,30 +209,14 @@
 					"command": "intuita.clearState",
 					"when": "false"
 				},
-				{
```

Comment:
should these commands be removed?

Reply:
yes, this is old code, we cannot accept/reject specific jobs or folders now